src/datamodel/serializer/Uuid.cpp

```diff
@@ -42,35 +30,38 @@ constexpr size_t LSB_LOW__ = 12;
 namespace uprotocol::datamodel::serializer::uuid {
 
 std::string AsString::serialize(const uprotocol::v1::UUID uuid) {
-	// Extracting the parts of the UUIDv8
-	uint64_t unix_ts_ms = (uuid.msb() >> TIMESTAMP_SHIFT) & TIMESTAMP_MASK;
-	uint8_t ver = (uuid.msb() >> VERSION_SHIFT) & VERSION_MASK;
-	uint16_t counter = uuid.msb() & COUNTER_MASK;
-	uint8_t var = (uuid.lsb() >> VARIANT_SHIFT) & VARIANT_MASK;
-	uint64_t rand_b = uuid.lsb() & RANDOM_MASK;
+	// Extracting the parts of the UUIDv7
+	uint64_t unix_ts_ms =
+	    (uuid.msb() >> UUID_TIMESTAMP_SHIFT) & UUID_TIMESTAMP_MASK;
+	uint8_t ver = (uuid.msb() >> UUID_VERSION_SHIFT) & UUID_VERSION_MASK;
+	uint16_t counter = uuid.msb() & UUID_COUNTER_MASK;
```

Comment:
Minor: Since UUIDv8 calls this field `rand_a`, can you change the name of the variables related to that field? I think it will help with reading this code against the UUID RFC in the future.